mkosi.build

```diff
@@ -259,3 +259,44 @@ fi
 # Make sure services aren't enabled by default on Debian/Ubuntu.
 mkdir -p "$DESTDIR/etc/systemd/system-preset"
 echo "disable *" > "$DESTDIR/etc/systemd/system-preset/99-mkosi.preset"
+
+if [ -d mkosi.kernel/ ]; then
+        cd "$SRCDIR/mkosi.kernel"
```

Comment:
doesn't matter, but maybe pushd instead of cd?

Reply:
This is a posix shell script actually, so we don't have pushd. We should probably make it bash, or maybe even python, but let's leave that for another PR